modules/integration_aws-alb/README.md

```diff
@@ -43,7 +43,7 @@ Note the following parameters:
   instance of the module.
   Its value will be added to the `prefixes` list at the start of the [detector 
   name](https://github.com/claranet/terraform-signalfx-detectors/wiki/Templating#example).
-  In general, it will also be used in `filter-tags` sub-module to apply a
+  In general, it will also be used in `filtering` internal sub-module to apply a
```

Comment:
I think this should be changed to either `apply a filter` or `apply filtering`.

Reply:
`apply a filter(s)` is good for you?

Reply:
Simply `apply filters` would be better.

Reply:
done !